lib/pptx/shapes/textbox.rb

```diff
@@ -46,7 +46,7 @@ def convert_string_to_jpg(string, font_size, size)
           </head>
           <body>#{string.strip.gsub(/\n/, '<br>').gsub("\"", "&quote;")}</body>|
 
-        system("echo \"#{html}\" | wkhtmltoimage --width #{size.floor} - #{path}")
+        system("echo '#{html}' | wkhtmltoimage --width #{size.floor} - #{path}")
```

Comment:
thanks for your fix :)
can it be that its still there !?
if i put something like 
```
'`echo ~/.ssh/ida_rsa`'
```
as input in fields?

Reply:
Nope. bash only interpolates in double-quoted strings, not in single-quoted ones.
That is why 
```
html = "<p> `rm -rf /` </p>"
system("echo \"#{html}\"")
```
is evil, but
```
html = <p> `rm -rf/` </p>"
system("echo '#{html}'")
```
is safe

Reply:
```suggestion
  file = Tempfile.new()
  file.write(html)
  system (wkhtmltoimage --width #{size.floor} #{file.path} #{path})
  file.close
  file.unlink
```

This is a pseudocode, somebody test